README-Hacking.md

```diff
@@ -72,10 +72,18 @@ button to expose the final two fields.
 
 ## WIP: Updating the Docker Image
 
-The [edxops/jenkins](https://hub.docker.com/r/edxops/jenkins/) image is used in the Docker steps above. If you'd like to
- add or modify Jenkins plugins, update `docker/plugins.txt`.
- 
-The required plugins have been pre-installed on the container. Feel free to install additional plugins, but
-if they are required for either other developers or deployment further steps will be required.  To add them to the 
-the Docker container, they must be listed in ```docker/plugins.txt```. To be installed in a deployment environment,
-they must be added to the appropriate Ansible role defaults.
+The [edxops/tools-jenkins](https://hub.docker.com/r/edxops/tools-jenkins/) image is used in the Docker steps above. The required plugins have been pre-installed on the container. Feel free to install additional plugins. If you'd like to add or modify Jenkins plugins, follow the steps below.
+
+Note: Adding or modifying Jenkins plugin(s) will wipe your Jenkins workspace.
+
+Update the plugin list by changing the plugin version number(s) or by adding additional plugin(s) in the tools_jenkins Ansible role in the configuration repository on your local machine. The role is available [here](https://github.com/edx/configuration/blob/master/playbooks/roles/tools_jenkins/defaults/main.yml). Build a new Docker image that incorporates your changes by running the following command from the root of the configuration repository:
+
+	$ docker build -f docker/build/tools-jenkins/Dockerfile -t edxops/tools-jenkins:latest .
```

Comment:
If make docker.build.tools_jenkins tags, then we should document that.  If it doesn't, then I'm good with this.  We'll clean that up later.